### ISSUE TITLE
Symbols in intermediate clauses crashes

### ISSUE THREAD

**Issue by japgolly** [liancheng/scalafix-organize-imports#22]
The majority of scalajs-react users will have this in their code:

```scala
import japgolly.scalajs.react.vdom.html_<^._
```

But it crashes with this library:

error:
```
scalafix.internal.v1.FileException: unexpected error processing file /home/golly/BeardedLogic/projects/shipreq/Code/webapp-client-project/src/main/scala/shipreq/webapp/client/project/lib/TextEditor.scala
Caused by: java.lang.RuntimeException: invalid symbol format
japgolly.scalajs.react.vdom.html_<^.
                                  ^
	at scala.sys.package$.error(package.scala:30)
	at scala.meta.internal.semanticdb.Scala$DescriptorParser.fail(Scala.scala:200)
	at scala.meta.internal.semanticdb.Scala$DescriptorParser.parseValue(Scala.scala:230)
	at scala.meta.internal.semanticdb.Scala$DescriptorParser.parseDescriptor(Scala.scala:252)
	at scala.meta.internal.semanticdb.Scala$DescriptorParser.entryPoint(Scala.scala:279)
	at scala.meta.internal.semanticdb.Scala$DescriptorParser$.apply(Scala.scala:287)
	at scala.meta.internal.semanticdb.Scala$ScalaSymbolOps.owner(Scala.scala:101)
	at scalafix.v1.Symbol.owner(Symbol.scala:18)
	at fix.OrganizeImports.toRef$1(OrganizeImports.scala:155)
	at fix.OrganizeImports.expandRelative(OrganizeImports.scala:159)
	at fix.OrganizeImports.$anonfun$organizeImports$2(OrganizeImports.scala:87)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:273)
	at scala.collection.immutable.List.foreach(List.scala:392)
	at scala.collection.TraversableLike.map(TraversableLike.scala:273)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:266)
	at scala.collection.immutable.List.map(List.scala:298)
	at fix.OrganizeImports.organizeImports(OrganizeImports.scala:87)
	at fix.OrganizeImports.fix(OrganizeImports.scala:60)
	at scalafix.internal.v1.Rules.$anonfun$semanticPatch$1(Rules.scala:75)
	at scala.collection.immutable.List.map(List.scala:286)
	at scalafix.internal.v1.Rules.semanticPatch(Rules.scala:73)
	at scalafix.internal.v1.MainOps$.unsafeHandleFile(MainOps.scala:234)
```

If you want to see real code you can try to run this over https://github.com/japgolly/scalajs-react

Thanks!

**Comment by liancheng** [comment]
Interesting... from the error message, it seems to be an issue from Scalameta. Let me find some time to investigate. Thanks for reporting, @japgolly!

**Comment by japgolly** [comment]
No worries and again: thanks so much for your work on this! It's fantastic!

**Comment by liancheng** [comment]
@japgolly, could you please help to construct a minimal project to reproduce this issue? I failed to reproduce this one with the following diff:

```diff
diff --git a/input/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala b/input/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala
new file mode 100644
index 0000000..9c31486
--- /dev/null
+++ b/input/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala
@@ -0,0 +1,10 @@
+/*
+rules = OrganizeImports
+OrganizeImports.expandRelative = true
+ */
+package fix
+
+import IdentWithSymbols._
+import html_<^._
+
+object ExpandRelativeIdentWithSymbols
diff --git a/output/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala b/output/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala
new file mode 100644
index 0000000..64f5c3a
--- /dev/null
+++ b/output/src/main/scala/fix/ExpandRelativeIdentWithSymbols.scala
@@ -0,0 +1,6 @@
+package fix
+
+import fix.IdentWithSymbols._
+import fix.IdentWithSymbols.html_<^._
+
+object ExpandRelativeIdentWithSymbols
diff --git a/shared/src/main/scala/fix/IdentWithSymbols.scala b/shared/src/main/scala/fix/IdentWithSymbols.scala
new file mode 100644
index 0000000..adeda4d
--- /dev/null
+++ b/shared/src/main/scala/fix/IdentWithSymbols.scala
@@ -0,0 +1,5 @@
+package fix
+
+object IdentWithSymbols {
+  object html_<^
+}

```

**Comment by liancheng** [comment]
@japgolly, BTW, which Scalafix version are you using?

**Comment by japgolly** [comment]
Hi @liancheng , as I was creating a reproduction for this I found the problem :) This is an issue with scalafix 0.9.14 but seems to be fixed in 0.9.15. So sorry to waste your time! Thank you
